src/App/DocumentObserverPython.h

```diff
@@ -83,8 +83,6 @@ class AppExport DocumentObserverPython
     void slotRedoDocument(const App::Document& Doc);
     /** Called when a given object is recomputed */
     void slotRecomputedObject(const App::DocumentObject& Obj);
-    /** Called before an observed document is recomputed */
-    void slotBeforeRecomputeDocument(const App::Document& Doc);

```

Comment:
Experimental code should not break the python API by removing features.

Reply:
There are many addons outside the freecad core using the python API, all over github and potentially even not made public but used for private / restricted use cases. This has to be considered.

Reply:
Assembly3 uses it and will be broken without: https://github.com/realthunder/FreeCAD_assembly3

> There could be some private addon using it, but this signal is incompatible with an async world.

But than there has to be a deliberate decision to break the python API and removing functionality. I don't like doing this just on the fly during development of a maybe feature without discussion or notification to anyone, no deprecation messages, nothing... Perfectly fine for a experimental branch, but not for merging into main.

Reply:
thanks!